lib/dfid-transition/patch/rummager/countries.rb

```diff
@@ -0,0 +1,37 @@
+require 'dfid-transition/patch/rummager/base'
+require 'govuk/registers/country'
+require 'rest-client'
+
+module DfidTransition
+  module Patch
+    module Rummager
+      class Countries < Base
+        def mutate_schema
+          add_country_field
+          add_search_terms
+        end
+
+      private
+
+        def add_country_field
+          schema_hash.fetch('fields') | ['country']
+        end
+
+        def add_search_terms
+          schema_hash
+            .fetch('expanded_search_result_fields')['country'] =
```

Comment:
Why is the method called `add_search_terms` when it replaces `expanded_search_result_fields/country`? Do we have a better handle on what the latter does now, and does `search_terms` reflect that purpose?